src/components/Inputs/GeojsonAttributePicker.js

```diff
@@ -0,0 +1,93 @@
+import i18n from '@dhis2/d2-i18n'
+import { ReactFinalForm, CheckboxFieldFF } from '@dhis2/ui'
+import PropTypes from 'prop-types'
+import React from 'react'
+import { ResourcePickerField } from '../index'
+import { resourceTypes } from '../ResourcePicker/resourceTypes'
+
+const { useField, Field } = ReactFinalForm
+
+const USE_ATTRIBUTE_NAME = 'useAttribute'
+const USE_ATTRIBUTE_LABEL = i18n.t('Import as associated geometry')
+const USE_ATTRIBUTE_DATATEST = 'input-use-attribute'
+
+const NAME = 'geojsonAttribute'
+const LABEL = i18n.t('Store geometry as value of attribute')
+const DATATEST = 'input-geojson-attribute-picker'
+const LISTNAME = 'geojsonAttribute'
+const FILTERLABEL = ''
+const SELECTEDLABEL = i18n.t('Selected programs')
+const ERRORMESSAGE = i18n.t('Something went wrong when loading the attributes!')
+const RESOURCETYPE = resourceTypes.GEOJSON_ATTRIBUTE
+
+const VALIDATOR = (selectedAttributes = []) =>
+    selectedAttributes.length === 0
+        ? i18n.t('One attribute must be selected')
+        : undefined
+
+const GeojsonAttributePicker = ({
```

Comment:
@HendrikThePendric Are all input components supposed to be located here, even if the component would most likely never be used by another importer?

Reply:
Yes

Reply:
I have moved the specific Geometry components to the components/Geometry folder to clean up a bit. Tell me if I should move them to the components/Inputs folder.